applications/RomApplication/python_scripts/calculate_rom_basis_output_process.py

```diff
@@ -120,6 +120,13 @@ def ExecuteFinalize(self):
         rom_basis_dict = {
             "train_hrom": False,
             "run_hrom": False,
+            "solving_strategy": "Galerkin",
```

Comment:
```suggestion
            "solving_strategy": "galerkin",
```
For the json I/O we use `snake_case`.

Reply:
Besides, I'd try to be consistent in the naming with the `hrom_training_utility.py`. Hence, I'd name this `projection_strategy`.

Reply:
I made this change.